src/functions/layer.js

```diff
@@ -85,8 +88,10 @@ export const setup = (props, leafletRef, context) => {
     },
   };
 
+  provide("bindPopup", methods.bindPopup);
   provide("bindTooltip", methods.bindTooltip);
   provide("unbindTooltip", methods.unbindTooltip);
+  provide("unbindPopup", methods.unbindPopup);
```

Comment:
```suggestion
  provide("bindPopup", methods.bindPopup);
  provide("unbindPopup", methods.unbindPopup);
  provide("bindTooltip", methods.bindTooltip);
  provide("unbindTooltip", methods.unbindTooltip);
```

Very minor point, but if the main motivation for the composition API is to allow code that works on a given feature to be more easily grouped together, we could arrange by feature (popup vs tooltip) instead of function (bind vs unbind). Just an idea, WDYT?